packages/rum-core/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -26,19 +26,7 @@ const FAKE_NAVIGATION_ENTRY: RumPerformanceNavigationTiming = {
   loadEventEnd: 567 as RelativeTime,
 }
 
-function spyOnViews() {
-  const handler = jasmine.createSpy()
-
-  function getViewEvent(index: number) {
-    return handler.calls.argsFor(index)[0] as ViewEvent
-  }
-
-  function getHandledCount() {
-    return handler.calls.count()
-  }
-
-  return { handler, getViewEvent, getHandledCount }
-}
+const configuration: Partial<Configuration> = { isEnabled: () => true }
```

Comment:
Same here, the `configuration` provided by the "setup builder" should be used instead